_posts/2016-05-05-clearing-the-air.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Clearing the Air about Vulnerabilities in OpenStack"
+date:   2016-05-05
+categories: vulnerabilities
+tags: security vmt vulnerabilities summit
+author: "tmcpeak @ IBM Cloud, sicarie @ HPE"
+---
+
+Recently, there have been a few talks around "vulnerabilities" within
+the OpenStack project that missed the mark so much they deserve note.
+
+Some of them call out important concepts such as
+[CIA](https://en.wikipedia.org/wiki/Information_security#Key_concepts)
+and the [CVE database](https://cve.mitre.org/). Unfortunately, they all
+attempt to highlight vulnerabilities or attack vectors within OpenStack
+that have either been addressed years ago, or are not able to be addressed
+by the upstream community and are the responsibility of the group
+deploying and maintaining the cloud.
+
+To understand how OpenStack handles vulnerabilities securely, it is
+important to briefly introduce the OpenStack vulnerability management
+process.
+
+## Vulnerability Management in OpenStack
+A vulnerability in OpenStack usually begins life as a bug filed against
+a project with the "security" tag.  These bugs are marked private and sent
+directly to the project's security team and the VMT.  An initial triage is
+performed to understand whether the bug represents a legitimate security issue
+and if so what the impact is.  If the issue is confirmed, an advisory and patch
+are prepared and validated privately.  Once the the advisory and fix are
+available, OpenStack stakeholders are given two weeks early notice to patch
```

Comment:
Should probably write out, that "distributions and vendors" (where the majority of deployers base their installations on)